src/controller.rs

```diff
@@ -53,6 +57,11 @@ impl mio::Handler for Controller {
         println!("Receiving a notification! {}", data.description());
 
         let mut context = self.context.lock().unwrap();
+
+        for socket in context.sockets.values() {
+            let _ = socket.send(data.description());
```

Comment:
can we at least log when sending fail? And think about the right thing to do (eg. closing the socket?)